### ISSUE TITLE
can be used with windows Phone 10

### ISSUE THREAD

**Issue by lse123** [floatinghotpot/cordova-plugin-flurry#38]
can be used with windows Phone 10 this plugin in XDK/Cordova apps???? Or deprcated like admob did for WP8/10 ads????

**Comment by floatinghotpot** [comment]
No, it cannot be used for windows, as flurry has no windows SDK.
